apps/demos/Demos/DataGrid/CustomEditors/Angular/app/app.component.ts

```diff
@@ -52,6 +52,10 @@ export class AppComponent {
       },
     });
   }
+  
+  getSelectedRowKeys(value: any): any[] {
```

Comment:
We try to avoid use the any in our public demos and examples :)
So, let's make this method with a generic type:

```suggestion
  getSelectedRowKeys<T>(value: T): T[] {
```